easybuild/easyconfigs/i/InterProScan/InterProScan-5.50-84.0-data.eb

```diff
@@ -0,0 +1,71 @@
+# Built with EasyBuild version 4.3.3-r3ff498d1f735c51b313c04c5e4abeeaecc23589b on 2021-03-05_23-15-26
+# Built with EasyBuild version 4.1.1-rf312b754b3a377fbd5057b6207f73845cab491ab on 2020-02-19_23-09-18
+easyblock = 'Tarball'
+
+name = 'InterProScan'
+version = '5.50-84.0'
+versionsuffix = '-data'
+homepage = 'http://www.ebi.ac.uk/interpro/'
+description = """This module containes only InterProScan data part. 
+InterProScan is a sequence analysis application (nucleotide and protein 
+sequences) that combines different protein signature recognition methods 
+into one resource."""
+
+
+toolchain = SYSTEM
+source_urls = [
+    'http://ftp.ebi.ac.uk/pub/software/unix/iprscan/%(version_major)s/%(version)s/',
+]
+sources = ['%(namelower)s-%(version)s-64-bit.tar.gz']
+checksums = ['3b792b33a14091a144a13ba35d70fbaa45abb8102f240a6da4e24bb07babaac4']
+
+dependencies = [
```

Comment:
I changed it to builddependencies as these dependencies are only needed during building this datasets.